lib/console.js

```diff
@@ -54,6 +55,8 @@ function Console(stdout, stderr, ignoreErrors = true) {
   prop.value = createWriteErrorHandler(stderr);
   Object.defineProperty(this, '_stderrErrorHandler', prop);
 
+  this[kCounts] = Object.create(null);
```

Comment:
The reason why I preferred a Map is, well, this object _is_ used as a hashmap. We should try to prevent depending on how V8 internally implements object lookup but try to make the code [explicit/declarative](http://benediktmeurer.de/2017/04/03/v8-behind-the-scenes-march-edition/) &ndash; especially when a Map provides [performance improvements](https://github.com/nodejs/node/pull/11930) over `Object.create(null)` too.